package.json

```diff
@@ -1,102 +1,107 @@
 {
   "name": "timed",
-  "version": "0.12.1",
+  "version": "0.13.0",
   "description": "A time tracking software written in Ember.js",
+  "repository": "https://github.com/adfinis-sygroup/timed-frontend",
+  "license": "AGPL-3.0",
+  "author": "Adfinis SyGroup AG <info@adfinis-sygroup.ch>",
   "directories": {
     "test": "tests"
   },
   "scripts": {
     "build": "ember build",
     "start": "ember server --proxy http://localhost:8000",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
     "test": "ember test"
   },
-  "repository": "https://github.com/adfinis-sygroup/timed-frontend",
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "10.* || >= 12.*"
   },
-  "author": "Adfinis SyGroup AG <info@adfinis-sygroup.ch>",
-  "license": "AGPL-3.0",
   "devDependencies": {
-    "@html-next/vertical-collection":
-      "github:html-next/vertical-collection#5c4fc6b18c75ec161eee39622dd0c068bff735f2",
-    "babel-eslint": "^8.0.1",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
-    "broccoli-asset-rev": "^2.4.5",
+    "@babel/plugin-proposal-decorators": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.8.3",
+    "@babel/plugin-transform-async-to-generator": "^7.8.3",
```

Comment:
This is included by `ember-cli-babel` - remove

Reply:
Thank you!